src/sql/workbench/services/objectExplorer/common/objectExplorerService.ts

```diff
@@ -237,7 +241,7 @@ export class ObjectExplorerService implements IObjectExplorerService {
 			if (this._sessions[session.sessionId]) {
 				connection = this._sessions[session.sessionId].connection;
 
-				if (session && session.success && session.rootNode) {
+				if (session.success && session.rootNode) {
```

Comment:
apparently we don't need to null check `session`?  though I agree even if we did this wouldn't be the correct place to check since we've already accessed it on line 241 above.